x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.test.tsx

```diff
@@ -20,7 +20,15 @@ import { useAllExceptionLists } from './use_all_exception_lists';
 jest.mock('../../../../../../common/lib/kibana');
 jest.mock('./use_all_exception_lists');
 jest.mock('../../../../../../shared_imports');
+jest.mock('@kbn/i18n/react', () => {
```

Comment:
This is also present a few times. Here are the errors we clear up:
```
      console.error
      [React Intl] Error formatting relative time.
      RangeError: The date value provided to IntlRelativeFormat#format() is not in valid range.

      at defaultErrorHandler (node_modules/react-intl/lib/index.js:525:13)
      at formatRelative (node_modules/react-intl/lib/index.js:748:5)
      at FormattedRelative.render (node_modules/react-intl/lib/index.js:1354:31)
      at finishClassComponent (node_modules/react-dom/cjs/react-dom.development.js:18470:31)
      at updateClassComponent (node_modules/react-dom/cjs/react-dom.development.js:18423:24)
      at beginWork$1 (node_modules/react-dom/cjs/react-dom.development.js:20186:16)
      at beginWork$$1 (node_modules/react-dom/cjs/react-dom.development.js:25756:14)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:24698:12)
```
```
      console.warn
      Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

      * Move data fetching code or side effects to componentDidUpdate.
      * If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
      * Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

      Please update the following components: FormattedRelative
```

Reply:
Hey, thanks for doing all of this! Just a quick question so I understand a bit better...isn't `FormattedRelative` implemented in the module already? What do we get by mocking it specifically here?

Reply:
what do you mean implemented in the module? Just mocking this component since we are not responsible for it, we are not testing anything to do with it, and it is causing errors in code.